modules/claims_api/app/swagger/claims_api/forms/veteran_identifier_example.json

```diff
@@ -0,0 +1,8 @@
+{
+  "attributes": {
```

Comment:
any chance we could at least consider not having this wrapped with both attributes and data?
im not even sure where its being wrapped by data actually...

Reply:
This one has me stumped.
I'm not sure where the requirement for the `data` -> `attributes` nesting structure is coming from.
It's not defined in my schema definition file.